.changeset/metal-hornets-travel.md

```diff
@@ -0,0 +1,16 @@
+---
+"@fluidframework/container-runtime": minor
```

Comment:
@tylerbutler should this say `patch` now?

Reply:
It won't be used regardless, so just leave it as is to make the cherry-pick clean. (We don't use changesets with patches. Maybe we should. Separate discussion.

Reply:
The cherry-pick is way not clean because of all the churn here.  But I can revert it when I take Jillian's suggestions

Reply:
It's great to have the changeset, even if unused in this branch.